samples/boards/nrf52/mesh/onoff_level_app/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "common.h"
+
+void main(void)
+{
+	int err, rc = 0;
+
+	rc = nvs_init(&fs, FLASH_DEV_NAME, STORAGE_MAGIC);
+
+	if (rc) {
+		printk("Flash Init failed\n");
+	}
+
+	if (nvs_read(&fs, NVS_LED_STATE_ID,
+		     &light_state_current,
+		     sizeof(struct light_state_t)) < 0) {
+
+		printk("\n\rentry for NVS_LED_STATE_ID is not found on NVS\n");
+
+		light_state_current.OnOff = 1;
+		light_state_current.power = LEVEL_100;
+	}
+
+	printk("\n\rled_on_off_status -> %d, led_level_status -> %d\n",
```

Comment:
still \r here in the beginning. Actually why do you start the format string with a new line? Isn't the one at the end sufficient?